tools/distrib/check_namespace_qualification.py

```diff
@@ -54,6 +55,9 @@ def check(self, fpath, fix):
             # skip `#define` statements
             if self.define_re.search(line):
                 continue
+            # skip if previous line is an incomplete using namespace statement
```

Comment:
Why is this necessary? There are ~60 spots in the src/ tree that have multi-line `using X = \n some::type;` statements, and this script hasn't flagged them.

Reply:
Ah interesting, the script doesn't like it when you fully quality. Weird.